woocommerce/woocommerce
# Standardize builds steps for WooCommerce Beta Tester

### All Submissions:






### Changes proposed in this Pull Request:

Similar to https://github.com/woocommerce/woocommerce/pull/32689, this PR standardises build steps such that they are isolated from other plugins or packages and do not produce a zip artefact.

The `build` step now only builds the minified files. Meanwhile, `build:zip` now builds and creates a zip. The `build_step` configuration option has been added so WooRelease knows how to build and create a zip.

While I was here, I noticed Jetpack Changelogger hadn't been added, so I've done that in this PR.

Partially fixes https://github.com/woocommerce/woocommerce/issues/32654

### How to test the changes in this Pull Request:

1. `pnpm install`
2. If https://github.com/woocommerce/woocommerce/pull/32657 has been merged and is present in this branch, skip this step. Otherwise, `cd plugins/woocommerce-beta-tester && composer install`
3. Delete built files `plugins/woocommerce-beta-tester/assets/js/version-information.min.js` and `plugins/woocommerce-beta-tester/assets/js/version-picker.min.js`.
4. `pnpm nx build woocommerce-beta-tester `
5. See the command succeed and the files you deleted in a previous step get built and reappear.

### Other information:




### FOR PR REVIEWER ONLY:
